docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -864,6 +866,11 @@ an ip address to `nginx.ingress.kubernetes.io/influxdb-host`. If you deploy Infl
 
 ### Backend Protocol
 
+!!! attention
+Be aware that many annotations described in this document assume HTTP or HTTPS
```

Comment:
This part is a bit confusing to me. Can you clarify an example of when server-snippet and backend-specific configs would be required?

Also per your comment on the PR about polymorphic behaviour I would love to see a detailed issue on that, it can be helpful for us to think if this is a feature we need and implement it :)

Reply:
> This part is a bit confusing to me. Can you clarify an example of when server-snippet and backend-specific configs would be required?

Should I include a example of how to achieve the equivalent settings for Custom Timeouts (with server-snippet) ? I'm not sure what's confusing exactly ?.

> Also per your comment on the PR about polymorphic behaviour I would love to see a detailed issue on that, it can be helpful for us to think if this is a feature we need and implement it :)

Sure, I'll do that.

Reply:
@rikatz Could you elaborate ? I'm still not sure on what is confusing.